index.html

```diff
@@ -0,0 +1,272 @@
+                ><img src="./assets/images/password.png" />
+              </a>
+            </div>
+            <!-- deployed link -->
+            <div>
+              <a class="button" href="">deployed link</a>
+            </div>
+          </div>
+
+          <!-- third project card -->
+
+          <div class="project-card">
+            <!-- third project card title  -->
+            <h3 class="project-title">Coding Quiz</h3>
+
+            <!-- third project paragraph  -->
+
+            <p class="project-info">
+              In this project the main task was to design a coded quiz.
+            </p>
+
+            <!-- third project image -->
+
+            <div class="project-image">
+              <a href="https://github.com/AOsman0/code-quiz/tree/dev"
+                ><img src="./assets/images/quiz.jpg"
+              /></a>
+            </div>
+            <!-- deployed link -->
+            <div>
+              <a class="button" href="">deployed link</a>
+            </div>
+          </div>
+
+          <!-- fourth project card -->
+
+          <div class="project-card">
+            <!-- fourth project title -->
+
+            <h3 class="project-title">Creating A Planner</h3>
+
+            <!-- fourth project info -->
+
+            <p class="project-info">
+              In this project the main task was to design a planner.
+            </p>
+
+            <!-- fourth project image -->
+
+            <div class="project-image">
+              <a href="https://github.com/AOsman0/day-planner/tree/dev">
+                <img class="images" src="./assets/images/planner.png" />
+              </a>
+            </div>
+            <!-- deployed link -->
+            <div>
+              <a class="button" href="">deployed link</a>
+            </div>
+          </div>
+          <!-- fifth project card -->
+
+          <div class="project-card">
+            <!-- fifth project title -->
+
+            <h3 class="project-title">Weather Dashboard</h3>
+
+            <!-- fifth project info -->
+
+            <p class="project-info">
+              The main task of this project is to create a weather dashboard.
+            </p>
+            <!-- fifth project image -->
+
+            <div class="project-image">
+              <a href="https://github.com/AOsman0/weather-dashboard/tree/dev"
+                ><img class="images" src="./assets/images/weather-dash.png" />
+              </a>
+            </div>
+            <!-- deployed link -->
+            <div>
+              <a class="button" href="">deployed link</a>
+            </div>
+          </div>
+
+          <!-- closing div tag of container and section -->
+        </div>
+      </section>
+      <hr />
+
+      <!-- Contact Me Section -->
+      <section id="contact">
+        <div>
+          <h2 class="contact-header">Contact Details</h2>
+        </div>
+        <!-- Contact Container -->
+        <div class="contact-container">
```

Comment:
It would of been nice if these were in a horizontal line, rather than vertical, but that's just me being picky!